setup.py

```diff
@@ -30,28 +30,32 @@
           ]},
           package_dir={"": "src"},
           install_requires=[
```

Comment:
Using fixed versions in setup.py makes it harder to install in an existing environment, and the onus is on you to keep this up-to-date regularly.